src/test/java/org/xmpp/packet/JIDEqualsHashCodeTest.java

```diff
@@ -41,6 +41,18 @@ protected Object createNotEqualInstance() throws Exception {
 		return new JID("edon@niamod/ecrouser");
 	}
 
+	public void testNullOtherResource() throws Exception {
+                // Setup fixture.

```

Comment:
Something's inconsistent with the indentation of this method as compared to other methods?

Reply:
the rest of the file is using naughty tabs